src/sidebar/services/annotations-exporter.tsx

```diff
@@ -233,12 +234,21 @@ export class AnnotationsExporter {
             {annotations.map((annotation, index) => {
               const page = pageLabel(annotation);
               const annotationQuote = quote(annotation);
+              const renderedComment = renderMathAndMarkdown(annotation.text);
+
+              // When the result of rendering the text's markdown is just a
+              // single paragraph, we fall back to the annotation text, to
+              // avoid extra margins added by some editors, like Google Docs
+              const comment =
+                renderedComment === `<p>${annotation.text}</p>`
+                  ? annotation.text
+                  : renderedComment;
```

Comment:
This could end up being brittle if the HTML rendering changes. On the other hand, it is very simple and easy to adapt if we need to.

Reply:
Also, if this breaks, the tests should catch it.